index.html

```diff
@@ -779,9 +788,32 @@ <h2 id="table-of-contents">Table of Contents</h2>
               Introduction
             </h2>
             <div datatype="rdf:HTML" property="schema:description">
-              ...
+              <p><em>This section is non-normative.</em></p>
+              <p>
+                In the Solid ecosystem Pods and clients are disconnected. This means clients are tight to data models. Furthermore, clients and thus Pods should be able to be connected in a network of knowledge. The connection is only achievable if the data itself is easy to be interconnected. 
```

Comment:
Find another way to express "disconnected". Perhaps you mean not strongly coupled or loosely coupled.

Use "storage" or "server" or both, or another term - which is defined https://solidproject.org/TR/protocol#storage - instead of "Pod" - which is undefined as far as technical definitions go (i.e., no Solid spec uses "Pod" at this point.)